app/controllers/cookie_policy_acceptances_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CookiePolicyAcceptancesController < ApplicationController
+  include CookiePolicyAcceptance
```

Comment:
Any reason why the module is included both here and in `ApplicationController`?

Reply:
@MarcoBgn this was a leftover of a refactoring, thanks for spotting it, removed!